mitosheet/src/components/import/FileBrowser/FileBrowser.tsx

```diff
@@ -109,7 +109,8 @@ function FileBrowser(props: FileBrowserProps): JSX.Element {
                 loadingFolder: true
             }
         })
-        const _pathContents = await props.mitoAPI.getPathContents(currPathParts);
+        const response = await props.mitoAPI.getPathContents(currPathParts);
+        const _pathContents = 'error' in response ? undefined : response.result;
```

Comment:
> If this were to actually error, it would have generated an execution error and the modal would have opened, so we don't do anything here besides turn off the loading indicator.

No, so this is actually different now! If there is an error, we always get it back from the API call. if it sets the error modal, then it will both set the error modal AND get returned to you. 

I think this is much more intuitive, as it lets you handle in-place errors anywhere you want them, and will make us moving away from the error modal much easier going forward.